src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -263,10 +217,7 @@ - (void)sendHeaders:(NSDictionary *)metadata {
 - (void)writeMessage:(NSData *)message withErrorHandler:(void (^)())errorHandler {
 
   __weak GRPCCall *weakSelf = self;
-  GRPCEventHandler resumingHandler = ^(grpc_event *event) {
-    if (event->data.write_accepted != GRPC_OP_OK) {
-      errorHandler();
-    }
+  void(^resumingHandler)(void) = ^{
     // Resume the request writer (even in the case of error).
     // TODO(jcanizales): No need to do it in the case of errors anymore?
```

Comment:
Update these comments to `// Resume the request writer.`, as `startBatch` won't call this block upon errors.